core/controller/src/main/scala/whisk/core/controller/RestAPIs.scala

```diff
@@ -173,8 +173,6 @@ protected[controller] class RestAPIVersion(apipath: String, apiversion: String)(
                     webexp.routes()
                 } ~ {
                     swaggerRoutes
-                } ~ options {
```

Comment:
if you're not restoring this somewhere you are breaking "experimental" - plus options on the main routes (which we agreed is a separate issue to address/fix), no?